app/commands/git/sync.rb

```diff
@@ -14,13 +14,6 @@ def filepath_in_diff?(filepath)
       end
     end
 
-    def commits_from_current_to_head
-      walker = Rugged::Walker.new(git_repo.send(:rugged_repo))
```

Comment:
Removing my name from the codebase! 🙂

Reply:
Makes sense. Should I make an effort of removing any actual usernames from the codebase?